content/blog/nats-recover-quorum.md

```diff
@@ -123,7 +123,7 @@ $ nats server report jetstream
 ╰────────────────────┴────────────────┴─────────┴───────────┴──────────┴───────┴────────┴──────┴─────────┴─────────╯
 
 
-WARNING: No cluster meta leader found. The cluster expects 6 nodes but only 3 responded. JetStream operation require at least 4 up nodes.
+WARNING: No cluster meta leader found. The cluster expects 6 nodes but only 3 responded. JetStream operation requires at least 4 up nodes.
```

Comment:
ditto

Reply:
Well, this is still grammatically incorrect. That warning should be updated 😉

Reply:
https://github.com/nats-io/natscli/pull/974